instrumentation/dubbo-rpc/src/main/java/brave/dubbo/rpc/TracingFilter.java

```diff
@@ -111,6 +115,63 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
     }
   }
 
+  static class FinishSpanCallback implements ResponseCallback {
```

Comment:
can you move this back to same place so easier to tell what has changed?

Reply:
> can you move this back to same place so easier to tell what has changed?

done